engine/src/main/java/org/hibernate/search/query/dsl/sort/SortDistanceFieldAndReferenceContext.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.query.dsl.sort;
+
+/**
+ * @author Emmanuel Bernard emmanuel@hibernate.org
```

Comment:
Javadoc? (I don't remember if we have Javadoc for each DSL interface in the other DSLs so feel free to ignore if we're inline with the other DSLs)